Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
       ast (~> 2.4.1)
-    pg (1.3.5)
+    pg (1.4.3)
     propshaft (0.6.4)
       actionpack (>= 7.0.0)
       activesupport (>= 7.0.0)
       rack
       railties (>= 7.0.0)
-    public_suffix (4.0.6)
-    puma (5.6.4)
+    public_suffix (5.0.0)
+    puma (5.6.5)
       nio4r (~> 2.0)
     racc (1.6.0)
-    rack (2.2.3)
-    rack-test (1.1.0)
-      rack (>= 1.0, < 3)
-    rails (7.0.2.3)
-      actioncable (= 7.0.2.3)
-      actionmailbox (= 7.0.2.3)
-      actionmailer (= 7.0.2.3)
-      actionpack (= 7.0.2.3)
-      actiontext (= 7.0.2.3)
-      actionview (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activestorage (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    rack (2.2.4)
+    rack-test (2.0.2)
+      rack (>= 1.3)
+    rails (7.0.4)
+      actioncable (= 7.0.4)
+      actionmailbox (= 7.0.4)
+      actionmailer (= 7.0.4)
+      actionpack (= 7.0.4)
+      actiontext (= 7.0.4)
+      actionview (= 7.0.4)
+      activejob (= 7.0.4)
+      activemodel (= 7.0.4)
+      activerecord (= 7.0.4)
+      activestorage (= 7.0.4)
+      activesupport (= 7.0.4)
       bundler (>= 1.15.0)
-      railties (= 7.0.2.3)
+      railties (= 7.0.4)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.4.2)
+    rails-html-sanitizer (1.4.3)
       loofah (~> 2.3)
-    railties (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    railties (7.0.4)
```

Comment:
### [railties](https://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 7.0.2.3 → 7.0.4

[_[change-log](https://github.com/rails/rails/blob/v7.0.4/railties/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v7.0.4/railties), [gem-diff](https://my.diffend.io/gems/railties/7.0.2.3/7.0.4)_]





<details>
<summary>Commits</summary>

A change of **358** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v7.0.2.3...v7.0.4).

These are the first 10 commits:
- (ae565e5) [Merge pull request #44641 from SkipKayhil/fix-rubocop-errors](https://github.com/rails/rails/commit/ae565e5980c07c72d78a4c742ce2b876370a4ea0)
- (dd494f1) [Merge pull request #44637 from fatkodima/remove_foreign_key-…](https://github.com/rails/rails/commit/dd494f199f70db1641d9cbfeeafeb9e79fdaea9b)
- (64ad40e) [Merge pull request #44387 from piecehealth/fix_activestorage…](https://github.com/rails/rails/commit/64ad40e888ed844fd6076efbf2ac194936aafc7d)
- (22141fa) [Make sure allowed image processing arguments are correctly l…](https://github.com/rails/rails/commit/22141fa26519ae002cca9144913cc03438b0af6d)
- (e3b67a0) [Merge pull request #44663 from acallaghan/master](https://github.com/rails/rails/commit/e3b67a0542c2287a69e75431db9b586543dea46a)
- (ba0fa9e) [Merge pull request #44660 from julianrubisch/patch-1](https://github.com/rails/rails/commit/ba0fa9eb2f01871d07ab033fb07f58b6571fa10a)
- (4990a08) [Merge pull request #44672 from duffuniverse/update-docs-reop…](https://github.com/rails/rails/commit/4990a08cdc138b955fa3913f076b6d5b2ad6ec69)
- (66cb353) [Merge pull request #44675 from ghousemohamed/fix-rdoc-highli…](https://github.com/rails/rails/commit/66cb3539f768e34293eb23fae291723000ed2afa)
- (a0d819c) [Merge pull request #44598 from araishikeiwai/patch-1](https://github.com/rails/rails/commit/a0d819c469db63401cc1afdc3c561604c29d1e5f)
- (a9f4982) [Merge pull request #44678 from ghousemohamed/improvements-to…](https://github.com/rails/rails/commit/a9f4982504315ea2a30e79d4042ef6f2ec2d36b7)


</details>